integration-tests/templates/expo-rn74-template/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "expo/tsconfig.base",
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
is this only relevant for the expo app?

Reply:
well, no. it's right that all templates needed it. I've added that flag to the others + improved a bit the file with some basic configurations